specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -417,7 +413,7 @@
    "metadata": {},
    "source": [
     "## Step 4: Compute heating degree days and cooling degree days\n",
-    "Here, a heating degree day (HDD) is calculated by computing how many degrees Fahrenheit **colder** the daily temperature is from a specified temperature threshold. A cooling degree day (CDD) is calculated by computing how many degrees **warmer** the daily temperature is from a specified temperature threshold. In the computation below, you can provide different thresholds for HDD and CDD based on your needs. "
+    "Degree days are [based on the assumption](https://www.weather.gov/key/climate_heat_cool) that when the outside temperature is 65°F, we don't need heating or cooling to be comfortable. However, you may wish to use a different threshold than 65°F. In the code below, a heating degree day (HDD) is calculated by computing how many degrees Fahrenheit **colder** the daily temperature is from a specified temperature threshold. A cooling degree day (CDD) is calculated by computing how many degrees **warmer** the daily temperature is from a specified temperature threshold. In the computation below, you can provide different thresholds for HDD and CDD based on your needs. "
```

Comment:
Just to further prime readers to expect the function to allow two different thresholds, maybe make this say: "However, you may wish to use a different threshold than 65°F - for example, you might want to assume that temperatures between 60-70°F won't require heating or cooling, and use degrees below 60°F for HDD< and degrees above 70°F for HDD)." 

You could put a line break before "In the code..." if you think the paragraph is getting too long w/ this change

Reply:
I like this addition @gdicecco!